assets/js/components/ExecutionResults/CheckResultDetail/FactValue.jsx

```diff
@@ -3,11 +3,13 @@ import React from 'react';
 import ObjectTree from '@components/ObjectTree';
 
 function FactValue({ className, data }) {
-  return typeof data === 'object' ? (
-    <ObjectTree className={className} data={data} />
-  ) : (
-    <span className={className}>{`${data}`}</span>
-  );
+  if (data === null) {
```

Comment:
Just curious, this is needed because `typeof null` is `object`, right? :smile: 

Besides this I am wondering whether it would make sense having this check in `ObjectTree` itself. wdyt?

Reply:
> Just curious, this is needed because typeof null is object, right? 😄
Yes, it is a bug in js that it won't be fixed in 10000 years.

> Besides this I am wondering whether it would make sense having this check in ObjectTree itself. wdyt?
We have the same fix in the ObjectTree. We need to have in both places because we can get a plain fact value, so we don't always use ObjectTree

Reply:
fine with that :wink: